scripts/bwctest-osd.sh

```diff
@@ -0,0 +1,180 @@
+#!/bin/bash
+
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+function usage() {
+    echo ""
+    echo "This script is used to run bwc tests on a remote OpenSearch/Dashboards cluster."
```

Comment:
Can we spell out `backwards compatibility` in the documentation? I think acronyms in code is fine, but I don't think we need to use them as much in documentation.